src/components/TopBar/TopBar.vue

```diff
@@ -101,7 +101,9 @@
 			:model="localMediaModel"
 			@open-breakout-rooms-editor="showBreakoutRoomsEditor = true" />
 
-		<CallButton class="top-bar__button" />
+		<CallButton :is-screensharing="!!localMediaModel.attributes.localScreen"
+			class="top-bar__button"
+			:class="{'dark-hover' : !!localMediaModel.attributes.localScreen }" />
```

Comment:
I'd suggest at this step to:
- apply `:data-theme-dark="isInCall"` to the TopBar root element :moon: 
- replace `&.in-call` with `&[data-theme-dark="true"]`
- get rid of `in-call`, `dark-hover` and `.top-bar__button` classes
- ensure all buttons (and nested) have type `tertiary` or `tertiary-no-background` (LocalMediaControls)